src/utils.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable max-len */
+const availableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const errorMessage = {
+  missingText: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  missingCase: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  unavailableCase: 'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+};
+
+function createErrorResponse(res, statusCode, errors) {
+  res.statusCode = statusCode;
+  res.statusMessage = 'Bad Request';
+  res.setHeader('Content-Type', 'application/json');
+  res.end(JSON.stringify({ errors }));
+}
```

Comment:
You can use this function in the successful cases too

```suggestion
function sendResponse(res, statusCode, statusText, payload) {
  res.statusCode = statusCode;
  res.statusMessage = statusText;
  res.setHeader('Content-Type', 'application/json');
  res.end(JSON.stringify(payload));
}
```